src/api/app/jobs/backend_measurements_job.rb

```diff
@@ -0,0 +1,17 @@
+BackendMeasurementsJob < ApplicationJob
+  queue_as :quick
+
+  def perform
+    return unless CONFIG['amqp_options']
+
+    RabbitmqBus.send_to_bus('metrics', "workers #{worker_status_fields}")
```

Comment:
I added the backend measurements metrics to a different job, mainly because:
- To avoid a huge job with many responsibilities
- In case we have some backend communication problem, it doesn't affect the other metrics
- Because the information about workers and jobs are high volatile and dynamic we should pull it as often as we can